exporter/trace/trace.go

```diff
@@ -44,23 +53,72 @@ func newTraceExporter(o *options) (*traceExporter, error) {
 		client:    client,
 		o:         o,
 	}
+	b := bundler.NewBundler((*contextAndSpans)(nil), func(bundle interface{}) {
+		ctxSpans := bundle.([]*contextAndSpans)
+		ctxToSpansMap := make(map[context.Context][]*tracepb.Span)
+		// upload spans with same context in batch
+		for _, cs := range ctxSpans {
+			ctxToSpansMap[cs.ctx] = append(ctxToSpansMap[cs.ctx], cs.spans...)
+		}
+		for ctx, spans := range ctxToSpansMap {
+			e.uploadFn(ctx, spans)
+		}
+	})
+	if o.BundleDelayThreshold > 0 {
+		b.DelayThreshold = o.BundleDelayThreshold
+	} else {
+		b.DelayThreshold = 2 * time.Second
+	}
+	if o.BundleCountThreshold > 0 {
+		b.BundleCountThreshold = o.BundleCountThreshold
+	} else {
+		b.BundleCountThreshold = 50
+	}
+	if o.NumberOfWorkers > 0 {
+		b.HandlerLimit = o.NumberOfWorkers
+	}
+	// The measured "bytes" are not really bytes, see exportReceiver.
+	b.BundleByteThreshold = b.BundleCountThreshold * 200
```

Comment:
This should be done right after setting `BundleCountThreshold`, to make sure they are set together.
Also, there is no `exportReceiver` in this repo?